src/graphql/validation/specified_rules.py

```diff
@@ -118,15 +118,15 @@
         PossibleFragmentSpreadsRule,
         NoFragmentCyclesRule,
         UniqueVariableNamesRule,
-        NoUndefinedVariablesRule,
-        NoUnusedVariablesRule,
+        # NoUndefinedVariablesRule,
+        # NoUnusedVariablesRule,
         KnownDirectivesRule,
         UniqueDirectivesPerLocationRule,
         KnownArgumentNamesRule,
         UniqueArgumentNamesRule,
         ValuesOfCorrectTypeRule,
         ProvidedRequiredArgumentsRule,
-        VariablesInAllowedPositionRule,
+        # VariablesInAllowedPositionRule,
```

Comment:
This should be reverted

Reply:
Yes, the default validation behavior should be the same as in GraphQL.js which reflects the official specs. If you want to omit certain validation rules, see #70.

Reply:
Sorry, this was more to demonstrate that these rules are quite expensive compared to the others.

I wonder if these could be separated into "essential" and "friendly". For example, I think `NoUnusedVariablesRule` helps a user develop a query, but may not be needed in a production environment where there isn't a developer to provide feedback too.